typings/schemas/uischema.d.ts

```diff
@@ -1,11 +1,30 @@
 /// <reference path="../../typings/ui-grid/ui-grid.d.ts"/>
 
+//rule
+interface IRule {
+    effect: RuleEffect;
+    condition:ICondition;
+}
+declare enum RuleEffect {
+    HIDE,SHOW,ENABLE,DISABLE
+}
+interface ICondition {
+    type:string; //nice to have
+}
+interface ILeafCondition extends ICondition {
+    scope: {
+        $ref: string;
+    }
+    value: any;
```

Comment:
This is the expected value for the condition to evaluate to true, right? Can we rename this to `expectedValue` then?